rusk-wallet/src/bin/interactive.rs

```diff
@@ -707,6 +711,20 @@ fn confirm(cmd: &Command) -> anyhow::Result<bool> {
             println!("   > Max fee = {} DUSK", Dusk::from(max_fee));
             prompt::ask_confirm()
         }
+        Command::MoonlightStake {
```

Comment:
this command does not belong in this PR, it has nothing to do with the deployment

Reply:
@miloszm Removed